templates/Dockerfile_webservice.template

```diff
@@ -34,6 +34,8 @@ RUN wget --no-verbose https://github.com/jwilder/dockerize/releases/download/$DO
 ADD config/init_webservice.sh .
 ADD config/init_migration.sh .
 
+ADD {{ GITHUB_APP_PRIVATE_KEY_FILE }} ./dockstore_github_app_private_key.pem
```

Comment:
@agduncan94  Please test what happens if someone forgets to upload `GITHUB_APP_PRIVATE_KEY_FILE` to the host or has a typo. I believe that this will result in an empty folder (docker assumes missing files are folders in some cases and will create them). 
Ideally, if someone answers the corresponding question, we should check that the file exists and is non-empty, failing if it is missing because I think that will be an easy mistake to make in production.

Reply:
Answering own question, this properly fails the build process, doesn't create an empty folder unlike mounts.